firebase.json

```diff
@@ -20,6 +20,14 @@
       "source" : "/latest-debug.min.js",
       "destination" : "/client/v1.0.1-rc1-debug.min.js",
       "type" : 302
+    },{
+      "source" : "/beta.min.js",
+      "destination" : "/client/v1.0.1-rc2.min.js",
+      "type" : 302
+    },{
+      "source" : "/beta-debug.min.js",
```

Comment:
new beta libs means you can test against `https://js.bloombox.cloud/beta.min.js` and `../beta-debug.min.js`